examples/queries/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "examples-queries"
+
+publish.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dev-dependencies]
+alloy-network.workspace = true
+alloy-primitives.workspace = true
+alloy-provider = { workspace = true, features = ["pubsub", "ws"] }
+alloy-rpc-types.workspace = true
+alloy-rpc-client.workspace = true
+alloy-transport-http.workspace = true
```

Comment:
Please use the `Alloy` namespace as introduced in https://github.com/alloy-rs/examples/pull/11/files (now in `main`)